app/models/user.rb

```diff
@@ -3,4 +3,6 @@ class User < ApplicationRecord
                        uniqueness: true
 
   has_secure_password
+
+  enum role: %w("default", "admin")
```

Comment:
I think hound is right. I think it should be `enum role: %w(default, admin)` or `enum role: ["default", "admin"]
%w(a b c d...) is defined as ["a", "b", "c", "d"...], so the extra quotes could end up as ""a"",...